sql/core/src/main/scala/org/apache/spark/sql/ExperimentalMethods.scala

```diff
@@ -37,7 +37,7 @@ import org.apache.spark.sql.catalyst.rules.Rule
 class ExperimentalMethods private[sql]() {
 
   /**
-   * Allows extra strategies to be injected into the query planner at runtime.  Note this API
```

Comment:
This isn't necessary

Reply:
I removed a blank.

Reply:
Right, I'm saying it doesn't help anything. I'd avoid changes that don't do anything.

Reply:
I reverted it.